src/v2/view-builder/views/ov/EnrollChannelPollDescriptionView.js

```diff
@@ -237,4 +252,25 @@ export default View.extend({
       return false;
     });
   },
-});
\ No newline at end of file
+});
+
+function getDeviceMap(appState) {
+  if (!appState) {
+    return null;
+  }
+
+  const contextualData = appState.get('currentAuthenticator').contextualData;
+  let deviceMap = {};
+  if (contextualData.samedevice && contextualData.samedevice?.setupOVUrl) {
```

Comment:
The `contextualData.samedevice &&` check is redundant because you already are using `.?` in the next statement. Same below in the `else if`.